### DISCUSSION TITLE
Pasting an image from Windows clipboard into a node's dialogue or details

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#968] [Ideas]
I can copy an image and paste it _as a node_ in a mindmap. A dialogue opens giving me a chance to name it and, if desired, put it in a folder of my choosing.

I cannot do this within a node's dialogue or details.  To put it into one of these frames, I first save the clipboard image to a file on my local drive, copy its file path, paste the file path into a browser (like Edge). Then I can copy the image from the browser and paste it into the dialogue or details frames (layout view). I find this quite time-consuming. 

I would like to be able to do inside these two states what I can do when creating a node from a clipboard image.

Let me know if I'm not explaining this clearly. I can create a video to demo what I'm talking about.

If it is understood, might there be a way to make it happen?

~ Bal

**Comment by euu2021** [comment]
You can try [this add-on](https://docs.freeplane.org/getting-started/Add-ons_(install).html#insert-inline-image)
